src/docs/product/performance/trace-view.mdx

```diff
@@ -12,36 +12,34 @@ If you’re interested in being an Early Adopter, you can turn your organization
 
-Currently, the trace view will only show up to 100 transactions from the trace. Additional transactions can be found through [Discover](/product/discover-queries).
+### Entry Points
+- [Issue Details](/product/issues/issue-details/): In the issue detail, click "View Full Trace" below the Event ID (paired with Quick Trace).
+- [Event Details](/product/performance/event-detail/): In an event detail, click the trace ID on the top right (paired with Quick Trace).
+- [Transaction Summary](/product/performance/transaction-summary/): In the transaction summary, click the trace ID in the event table.
+- [Query Builder](/product/discover-queries/query-builder/): In Discover, build a query with the trace column, then click the trace ID in the resulting event table.
 
-## Trace View
+[Trace view image coming soon]
 
-The trace view provides a split view. On the left, you can view the trace's transaction tree. On the right, you can view a representation of each transaction as a colored rectangle. The first row lists the trace itself, with all other spans descending from that trace.
+At the top, you'll find a breakdown of the number of transactions and errors in this trace and how many projects they span across. You'll also get the total duration for this trace from start to end.
 
-You can display these views from:
-- [Transaction Summary](/product/performance/transaction-summary/): Select the [Performance Homepage](/product/performance/), the affected transaction, then click a trace ID.
-- [Event Details](/product/performance/event-detail/): Click the trace ID beneath the quick trace on the top right.
-- [Issue Details](/product/issues/issue-details/): Click "View Full Trace" above the quick trace.
-- [Query Builder](/product/discover-queries/query-builder/): Build a query with the trace column, then click a trace ID in the table.
+### Transaction View
 
-Trace IDs are linked to the corresponding Trace Detail.
+The transaction view provides a split view similar to the [span view](/product/performance/event-detail/#span-view). On the left, you can view the trace's transaction tree. On the right, you can view a representation of each transaction as a colored rectangle. The first row lists the trace itself, with all other spans descending from that trace. Clicking on any of the other rows will provide more detail about that transaction, including tags and other field data. You can either access more details about that [specific event](/product/performance/event-detail/) by clicking "View Transaction" or get a [transaction summary](/product/performance/transaction-summary/) by clicking "View Summary".
 
-## Transaction Details
+Note this view will only show up to 100 transactions from the trace. Additional transactions can be found through [Discover](/product/discover-queries).
```

Comment:
Should this have some special styling to make it visually distinct from the rest of the content?

Reply:
I can add an alert?

Reply:
I moved this into an alert